frontend/lib/ui/artefact_dialog/artefact_dialog.dart

```diff
@@ -35,41 +35,41 @@ class ArtefactDialog extends ConsumerWidget {
   Widget build(BuildContext context, WidgetRef ref) {
     final family = AppRoutes.familyFromContext(context);
     final artefacts = ref.watch(familyArtefactsProvider(family));
-    return artefacts.when(
-      data: (artefacts) {
-        final artefact = artefacts[artefactId];
-        return SelectionArea(
-          child: Dialog(
-            child: SizedBox(
-              height: min(800, MediaQuery.of(context).size.height * 0.8),
-              width: min(1200, MediaQuery.of(context).size.width * 0.8),
-              child: Padding(
-                padding: const EdgeInsets.symmetric(
-                  horizontal: Spacing.level5,
-                  vertical: Spacing.level3,
-                ),
-                child: (artefact == null)
-                    ? _invalidArtefactErrorMessage
-                    : Column(
-                        crossAxisAlignment: CrossAxisAlignment.start,
-                        children: [
-                          ArtefactDialogHeader(artefact: artefact),
-                          const SizedBox(height: Spacing.level4),
-                          ArtefactDialogInfoSection(artefact: artefact),
-                          const SizedBox(height: Spacing.level4),
-                          Expanded(
-                            child: ArtefactDialogBody(artefact: artefact),
-                          ),
-                        ],
-                      ),
-              ),
+    return SelectionArea(
+      child: Dialog(
+        child: SizedBox(
+          height: min(800, MediaQuery.of(context).size.height * 0.8),
+          width: min(1200, MediaQuery.of(context).size.width * 0.8),
+          child: Padding(
+            padding: const EdgeInsets.symmetric(
+              horizontal: Spacing.level5,
+              vertical: Spacing.level3,
+            ),
+            child: artefacts.when(
+              data: (artefacts) {
+                final artefact = artefacts[artefactId];
+                if (artefact == null) return _invalidArtefactErrorMessage;
+                return Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    ArtefactDialogHeader(artefact: artefact),
+                    const SizedBox(height: Spacing.level4),
+                    ArtefactDialogInfoSection(artefact: artefact),
+                    const SizedBox(height: Spacing.level4),
+                    Expanded(
+                      child: ArtefactDialogBody(artefact: artefact),
+                    ),
+                  ],
+                );
```

Comment:
Glad it looks neater. I actually made this movement because it used to show two loading indicators above each other when you open an artefact dialog directly through the url